packages/theme/src/__tests__/__snapshots__/theme.test.tsx.snap

```diff
@@ -69,6 +75,17 @@ Object {
     "colorIconWarning": "#D97706",
     "colorIconWeak": "#64748B",
     "colorIconWeaker": "#94A3B8",
+    "colorMapIcon": "#4A63FC",
+    "colorPrimary": "#102EE9",
+    "colorPrimaryButtonBackground": "#102EE9",
+    "colorPrimaryButtonBackgroundHover": "#041162",
+    "colorPrimaryButtonBackgroundOutlineFocus": "#102EE9",
+    "colorPrimaryButtonText": "#FFFFFF",
+    "colorProgressBarBackgroundPrimary": "#FFFFFF",
+    "colorProgressBarIndicatorPrimary": "#102EE9",
+    "colorSecondary": "#76FAC1",
+    "colorStepsMapBackground": "#102EE9",
```

Comment:
so maybe also remove steps here? 🤔 
```suggestion
    "colorMapBackground": "#102EE9",
```

Reply:
hmmm this is just a test snapshot and should reflect the json 🤔. Something is off!